xml-parser/src/test/java/org/uniprot/core/xml/unirule/ConditionSetConverterTest.java

```diff
@@ -0,0 +1,25 @@
+package org.uniprot.core.xml.unirule;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.uniprot.core.xml.AbstractConverterTest;
+import org.uniprot.core.xml.jaxb.unirule.ConditionSetType;
+
+public class ConditionSetConverterTest extends AbstractConverterTest {
+    public static ConditionSetType createObject() {
+        ConditionSetType conditionSetType =
+                objectCreator.createLoremIpsumObject(ConditionSetType.class);
+        conditionSetType.getCondition().addAll(ConditionConverterTest.createObjects());
+        return conditionSetType;
+    }
+
+    public static List<ConditionSetType> createObjects() {
+        return objectCreator.createLoremIpsumObject(ConditionSetTypeList.class).stream()
+                .filter(obj -> obj.getCondition().addAll(ConditionConverterTest.createObjects()))
```

Comment:
This is not filtering anything.... can you please use foreach like you do with the others.. it is easier to read